hunts/templates/tools.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+{% block page_content %}
+
+<div class="row justify-content-between" style="margin:10">
+    <div class="col-10">
+        <h1>Tools and References</h1>
+    </div>
+
+    <div class="col-10">
+        <h2>Word Puzzles</h2>
+    </div>
+    <div class="col-10">
+        <ul>
+            <li>https://quipqiup.com/</li>
+            <li>https://nutrimatic.org/</li>
+            <li>https://www.onelook.com/</li>
+            <li>https://www.thewordfinder.com/anagram-solver/</li>
+            <li>https://www.highlightpress.com.au/</li>
+            <li>http://www.npinsker.me/puzzles/crossword</li>
```

Comment:
Can we include a description, either as the link text or before or after the link? For example, it is important to note that Nathan's link doesn't help with crossword clues, but parses crosswords